src/main/java/eod/warObject/character/concrete/red/GuerrillaShooter.java

```diff
@@ -36,16 +37,16 @@ public String getName() {
     }
 
     @Override
-    public void attack() {
-        super.attack();
-        ArrayList<Damageable> affected = RequestRegionalAttack(player, attack).from(this).to(getAttackRange(), 1).getAffected();
+    public void attack(EffectExecutor executor) {
+        super.attack(executor);
+        ArrayList<Damageable> affected = RequestRegionalAttack(attack).from(this).to(player, getAttackRange(), 1).getAffected();
```

Comment:
affected should contain nothing cuz RegionalAttack.to() doesn't *really* attack the targets.

Reply:
fixed